### ISSUE TITLE
Fields with semicolons

### ISSUE THREAD

**Issue by jamesdeluk** [kerams/memrise-scraper#4]
The csv fields are split with semicolons, although if the lesson has a semicolon in the field, it will mess up the fields.

Is it possible to either:

1. Let the user decide the csv separator (e.g. tab, |)
2. Put all values in "quotes"

**Comment by kerams** [comment]
Yeah, I've forgotten about attribute and column names. Semicolons are replaced in level names, column and attribute values with |. Think I'll go for proper escaping with double quotes where necessary as it's the simples solution.

**Comment by jamesdeluk** [comment]
That would be awesome :)

As I'm here, some feature ideas:

1. Allow multiple to be downloaded at once to individual files e.g. `MemriseScraper.exe 1234 4567 7890`
2. Allow multiple to be downloaded and merged into single file e.g. `MemriseScraper.exe -Merge 1234 4567 7890` [keep headers row only from first course]

Really is a great tool, thanks for making it!

**Comment by jamesdeluk** [comment]
Actually, you know what - I take it back (partially)!

The course in question was https://app.memrise.com/course/278307/howtostudykoreancom-unit-3-2/6/

What was scraped was `Lesson 56;끼다;;to put on (a ring, gloves, sunglasses);;;gloves;sunglasses;etc…)`

And the actual record was:
![image](https://user-images.githubusercontent.com/27723508/109053632-a83e8980-7728-11eb-9d57-962d454dd364.png)

So the mistake is with the course, mostly, although this showed up because of the attributes.

**Comment by kerams** [comment]
Yeah, that course has very poor formatting. In any case, I've reimplemented escaping and athe new version is up. This is what the output looks like now [278307_2021-02-24.zip](https://github.com/kerams/memrise-scraper/files/6038400/278307_2021-02-24.zip). A couple of lines have double quotes, which weren't escaped before at all. Let me know if it's OK for you now.

Your suggestions are interesting and I might implement them when I feel bored ;).
